kombu/transport/pykafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.pykafka
```

Comment:
Oh, so you didn't just drop the hammer on my comment for an unknown reason.  What I commented on was that from out of left field came a high performance *third* [python client for kafka](https://github.com/confluentinc/confluent-kafka-python) and a blog post that benchmarked the three solutions:  http://activisiongamescience.github.io/2016/06/15/Kafka-Client-Benchmarking/